apps/admin_app/lib/admin_app_web/views/product_view.ex

```diff
@@ -36,6 +49,16 @@ defmodule AdminAppWeb.ProductView do
     length(product.taxon.variation_themes) > 0
   end
 
+  def is_variant_tracking_enabled(product) do
+    product.inventory_tracking == :variant
+  end
+
+  defp is_child_product(product) do
+    query = from(c in Variation, where: c.child_product_id == ^product.id)
+    count = Repo.aggregate(query, :count, :id)
+    count > 0
+  end
+
   def has_variants(product) do
```

Comment:
@SagarKarwande are you sure about placing this method in the `product_view`? can you justify the responsibility of the file and the this method?

Reply:
This method was included in the merge commit, while I was resolving the conflicts after merging the `develop` branch. 

This method is already refactored in develop branch: [Develop link](https://github.com/aviacommerce/avia/blob/4809919dcb389880d626c8edc485a21280a69a39/apps/admin_app/lib/admin_app_web/views/product_view.ex#L27)

I'll remove the duplicate entry of this method.

Reply:
This method can be moved to `Product` model.